tutorials/models/3_generative_model/5_dgmg.py

```diff
@@ -10,31 +10,36 @@
 
 ##############################################################################
 #
-# In earlier tutorials we have seen how learned embedding of a graph and/or
-# a node allow applications such as `semi-supervised classification for nodes
+# In this tutorial, you learn how to train and generate one graph at
+# a time. You also explore parallelism within the graph embedding operation, which is an
+# essential building block. The tutorial ends with a simple optimization that
+# delivers double the speed by batching across graphs.
+#
+# Earlier tutorials showed how embedding a graph or
+# a node enables you to work on tasks such as `semi-supervised classification for nodes
 # <http://docs.dgl.ai/tutorials/models/1_gcn.html#sphx-glr-tutorials-models-1-gcn-py>`__
 # or `sentiment analysis
 # <http://docs.dgl.ai/tutorials/models/3_tree-lstm.html#sphx-glr-tutorials-models-3-tree-lstm-py>`__.
 # Wouldn't it be interesting to predict the future evolution of the graph and
 # perform the analysis iteratively?
 #
-# We will need to generate a variety of graph samples, in other words, we need
-# **generative models** of graphs. Instead of and/or in addition to learning
-# node and edge features, we want to model the distribution of arbitrary graphs.
+# To address the evolution of the graphs, you generate a variety of graph samples. In other words, you need
+# **generative models** of graphs. Instead of and/or, in-addition to learning
+# node and edge features, you would need to model the distribution of arbitrary graphs.

```

Comment:
I believe we should just remove the "and/or, in-addition to" thing.  That would just be "Instead of learning nodes and edge features, you would need to ...".

I think you also mentioned the wrong Li; should be this one @mufeili .